File: Javascript/Exercicio06.js
const prompt = require('prompt-sync')();

var valores = new Array(5);
var soma = 0;

for (let i = 0; i < valores.length; i++) {
    numero = prompt("Informe um valor: ");
    valores[i] = parseFloat(numero);
    soma = soma + valores[i];
}
console.log("\nSoma: "+soma);

File: Java/Exercicio06.java
package Java;

import java.util.Scanner;

public class Exercicio06 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int [] valores = new int[5];
        int soma = 0;
        for (int i = 0; i < valores.length; i++) {
            System.out.print("Informe um valor: ");
            valores[i] = scanner.nextInt();
            soma = soma +valores[i];
        }
        System.out.println("\nSoma: "+soma);

        scanner.close();
    }
}
